src/libs/ajax/file-browser-providers/AzureBlobStorageFileBrowserProvider.ts

```diff
@@ -11,7 +11,7 @@ export interface AzureBlobStorageFileBrowserProviderParams {
 }
 
 type AzureBlobStorageFileBrowserProviderGetPageParams<T> = {
-  isFirstPage: boolean
+  isFirstPage?: boolean
```

Comment:
There's probably somewhere in here that needs to be changed to set a default value for `isFirstPage` to accommodate this change.